### ISSUE TITLE
Nightly crashes after enabling Server in config.xml

### ISSUE THREAD

**Issue by JanK118** [Vocaluxe/Vocaluxe#467]
Nightly crashes after enabling Server in config.xml

### Steps to reproduce
I enabled "Server Active" without encryption. After starting up Vocaluxe, I headed to the server-adress in the settings and scanned the QR Code with my smartphone. Vocaluxe crashed 20 seconds after. 
Now, it is crashing at startup. 
### Vocaluxe version and logfile
Vocaluxe 'Blue Sunrise' 0.4.0 (x64) (0.4.0-451-g805173f)

Vocaluxe.log:
[00:00:00.0379642] [Information] Starting to log
{ThreadId: 1, AdditionalData: ["{ Version = Vocaluxe 'Blue Sunrise' 0.4.0 (x64) (0.4.0-451-g805173f) }", "{ StartDate = 27.01.2020 01:33:32 }", "{ Id = 78917946-7857-401d-aeee-1957ce8a3282 }"], CallingContext: "{ callerMethodeName = Init, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CLog.cs, callerLineNumer = 128 }"}
[00:00:00.0689366] [Information] Started "Init Program"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.0739306] [Information] Started "Init Language"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.0979104] [Information] Finished "Init Language" successfully in 23.85ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.0989082] [Information] Started "Init Config"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.1408707] [Information] Finished "Init Config" successfully in 42.76ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.2112941] [Information] Started "Init Draw"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.3305222] [Information] Finished "Init Draw" successfully in 119.65ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.3315225] [Information] Started "Init Playback"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.4094512] [Information] Finished "Init Playback" successfully in 78.05ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.4094512] [Information] Started "Init Record"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.5452251] [Information] Finished "Init Record" successfully in 135.60ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.5462242] [Information] Started "Init Videodecoder"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.5472244] [Information] Finished "Init Videodecoder" successfully in 1.48ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.5472244] [Information] Started "Init Database"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.6790990] [Information] Finished "Init Database" successfully in 131.68ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.6790990] [Information] Started "Init Webcam"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.7160773] [Information] Finished "Init Webcam" successfully in 36.88ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.7170774] [Information] Started "Init Background Music"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.7210728] [Information] Finished "Init Background Music" successfully in 4.30ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.7210728] [Information] Started "Init Profiles"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.7240697] [Information] Finished "Init Profiles" successfully in 3.36ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:00.7250721] [Information] Started "Init Fonts"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:00.7340592] [Information] Started "Build Glyphs"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.1211757] [Information] Finished "Build Glyphs" successfully in 387.17ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.1211757] [Information] Finished "Init Fonts" successfully in 396.27ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.1211757] [Information] Started "Init Theme"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.1431547] [Information] Finished "Init Theme" successfully in 21.93ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.1431547] [Information] Started "Load Theme"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.5657655] [Information] Finished "Load Theme" successfully in 422.68ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.5657655] [Information] Started "Init Cover"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6187140] [Information] Finished "Init Cover" successfully in 52.38ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.6187140] [Information] Started "Init Screens"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6217111] [Information] Started "Build Screen List"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6327022] [Information] Finished "Build Screen List" successfully in 11.15ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.6327022] [Information] Started "Load Theme"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6386962] [Information] Started "Load Theme Test"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6876502] [Information] Finished "Load Theme Test" successfully in 49.36ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.6876502] [Information] Started "Load Theme Load"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.6896489] [Information] Finished "Load Theme Load" successfully in 2.27ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.6896489] [Information] Started "Load Theme Main"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.7206222] [Information] Finished "Load Theme Main" successfully in 30.12ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.7206222] [Information] Started "Load Theme Song"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.7585860] [Information] Finished "Load Theme Song" successfully in 38.11ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.7585860] [Information] Started "Load Theme Options"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.7605865] [Information] Finished "Load Theme Options" successfully in 1.97ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.7605865] [Information] Started "Load Theme Sing"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8015510] [Information] Finished "Load Theme Sing" successfully in 40.51ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8015510] [Information] Started "Load Theme Profiles"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8075405] [Information] Finished "Load Theme Profiles" successfully in 6.02ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8075405] [Information] Started "Load Theme Score"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8175318] [Information] Finished "Load Theme Score" successfully in 10.12ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8175318] [Information] Started "Load Theme Highscore"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8235265] [Information] Finished "Load Theme Highscore" successfully in 5.09ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8235265] [Information] Started "Load Theme OptionsGame"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8355145] [Information] Finished "Load Theme OptionsGame" successfully in 11.81ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8355145] [Information] Started "Load Theme OptionsSound"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8425090] [Information] Finished "Load Theme OptionsSound" successfully in 7.24ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8425090] [Information] Started "Load Theme OptionsRecord"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8534985] [Information] Finished "Load Theme OptionsRecord" successfully in 11.20ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8534985] [Information] Started "Load Theme OptionsVideo"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8584935] [Information] Finished "Load Theme OptionsVideo" successfully in 4.05ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8584935] [Information] Started "Load Theme OptionsVideoAdjustments"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8604943] [Information] Finished "Load Theme OptionsVideoAdjustments" successfully in 2.06ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8604943] [Information] Started "Load Theme OptionsLyrics"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8634888] [Information] Finished "Load Theme OptionsLyrics" successfully in 2.84ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8634888] [Information] Started "Load Theme OptionsTheme"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8704826] [Information] Finished "Load Theme OptionsTheme" successfully in 7.25ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8704826] [Information] Started "Load Theme Names"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.8784736] [Information] Finished "Load Theme Names" successfully in 7.58ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.8784736] [Information] Started "Load Theme Credits"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.9534038] [Information] Finished "Load Theme Credits" successfully in 74.42ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.9534038] [Information] Started "Load Theme Party"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:01.9544037] [Information] Finished "Load Theme Party" successfully in 1.69ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.9584002] [Information] Finished "Load Theme" successfully in 325.74ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.9584002] [Information] Finished "Init Screens" successfully in 340.13ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:01.9584002] [Information] Started "Init Server"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:02.2517282] [Information] Finished "Init Server" successfully in 293.19ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:02.2517282] [Information] Started "Init Input"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:02.2657151] [Information] Finished "Init Input" successfully in 13.24ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:02.2657151] [Information] Started "Init Game"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:02.3136725] [Information] Finished "Init Game" successfully in 48.61ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:02.3146718] [Information] Started "Init Party Modes"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:03.9095273] [Information] Finished "Init Party Modes" successfully in 1,595.17ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:03.9185215] [Information] Finished "Init Program" successfully in 3,849.82ms
{ThreadId: 1, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.0057364] [Information] Started "Loaded Songs Full"
{ThreadId: 1, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.0207229] [Information] Started "Load Songs"
{ThreadId: 15, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.0207229] [Information] Started "List Songs"
{ThreadId: 15, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.0217210] [Information] Finished "List Songs" successfully in 0.68ms
{ThreadId: 15, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.0217210] [Information] Started "Read TXTs"
{ThreadId: 15, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.1775815] [Information] Finished "Read TXTs" successfully in 155.49ms
{ThreadId: 15, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.1775815] [Information] Started "Sorted Songs"
{ThreadId: 15, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.1775815] [Information] Finished "Sorted Songs" successfully in 0.62ms
{ThreadId: 15, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.1805753] [Information] Finished "Load Songs" successfully in 160.11ms
{ThreadId: 15, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.2015539] [Information] Started "Loaded Covers"
{ThreadId: 16, CallingContext: "{ callerMethodeName = Start, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 57 }"}
[00:00:04.2844770] [Information] Finished "Loaded Covers" successfully in 83.71ms
{ThreadId: 16, CallingContext: "{ callerMethodeName = Stop, callerFilePath = C:\projects\vocaluxe\VocaluxeLib\Log\CBenchmark.cs, callerLineNumer = 74 }"}
[00:00:04.8383523] [Fatal] Unhandled error: "Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt."
{ThreadId: 1, CallingContext: "{ callerMethodeName = Main, callerFilePath = C:\projects\vocaluxe\Vocaluxe\Program.cs, callerLineNumer = 76 }"}
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei Vocaluxe.Base.Server.CVocaluxeServer.ProcessServerTasks() in C:\projects\vocaluxe\Vocaluxe\Base\Server\CVocaluxeServer.cs:Zeile 207.
   bei Vocaluxe.Lib.Draw.CDrawBase`1.MainLoop() in C:\projects\vocaluxe\Vocaluxe\Lib\Draw\CDrawBase.cs:Zeile 394.
   bei Vocaluxe.Lib.Draw.CDrawBaseWindows`1.MainLoop() in C:\projects\vocaluxe\Vocaluxe\Lib\Draw\CDrawBaseWindows.cs:Zeile 217.
   bei Vocaluxe.CMainProgram._Run(String[] args) in C:\projects\vocaluxe\Vocaluxe\Program.cs:Zeile 299.
   bei Vocaluxe.CMainProgram.Main(String[] args) in C:\projects\vocaluxe\Vocaluxe\Program.cs:Zeile 72.

**Comment by JanK118** [comment]
Enabling encryption made it worse, but deactivating the server completely let's me start the game again.

**Comment by Stefan1200-de** [comment]
Well, on my PC I'm able to start the web server without encryption without problems, I just had to allow the Windows UAC message.

Enabling the encryption raises this error messages in an endless loop:
![image](https://user-images.githubusercontent.com/15168351/73152985-5a44f580-40d2-11ea-8609-53686ff17158.png)
![image](https://user-images.githubusercontent.com/15168351/73153012-73e63d00-40d2-11ea-9b8b-2c1702b674df.png)

Vocaluxe\Base\Server\CVocaluxeServer.cs:Zeile 205/207
`Task task = _ServerTaskQueue.Dequeue();
task.RunSynchronously(TaskScheduler.FromCurrentSynchronizationContext());`

I don't know why you get this error, because I currently don't know the web server code: 
> System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.

Can you try a fresh installation of the nightly, just add the song folder and webserver setting on (without encryption) to the Config.xml file, and allow the Windows UAC message, if it happens again?

**Comment by lukeIam** [comment]
> Enabling the encryption raises this error messages in an endless loop:
> ![image](https://user-images.githubusercontent.com/15168351/73152985-5a44f580-40d2-11ea-8609-53686ff17158.png)

looks like the Security.Cryptography.dll is not found or has the wrong version in WebserverInitalConfig -> creates registers the certificates 
It's a separate executable to allow Vocaluxe run without UAC admin rights.
https://github.com/MicrosoftArchive/clrsecurity is the used library but it's quit old and may should be replaced...

**Comment by Stefan1200-de** [comment]
> looks like the Security.Cryptography.dll is not found or has the wrong version in WebserverInitalConfig -> creates registers the certificates
> It's a separate executable to allow Vocaluxe run without UAC admin rights.
> https://github.com/MicrosoftArchive/clrsecurity is the used library but it's quit old and may should be replaced...

Well, that stuff is not my specialism. And this clrsecurity project seems to be abandoned. But in my opinion within the local network the encryption of the website is not the biggest topic of Vocaluxe. I would guess it has time, if you has more time someday.

**Comment by JanK118** [comment]
>Well, on my PC I'm able to start the web server without encryption without problems, I just had to allow the Windows UAC message. ...

@Stefan1200-de Hi, sorry for the late response. 
Yeah, this error loop was what I meant with “made it worse”. 
I did this on the (to this date) freshest nightly with only two songs added in ...\songs and of course I allowed the UAC message. 
The problem occurred not exactly when I started Vocaluxe, but as I tried to connect to the server via my iPhone. I scanned the QR code, followed the link and it crashed. 
I’ll have a look at it this evening, maybe my home network or the iPhone was the problem, I’ll try it with some other devices. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Stefan1200-de** [comment]
Well, Vocaluxe should not crash, because someone is opening a link. But it would be nice to have more details what exactly you did and what kind of link was behind that QR code. If I'm able to reproduce this bug, I might be able to fix it. But I'm still not so deep into every parts of Vocaluxe to know where I should start searching, if I can't reproduce it.

**Comment by JanK118** [comment]
Well, I am sorry to even have bothered you. 
Nothing crashes today. Everything works as intended. I took the same version, tried it with the local browser and 127.0.0.1 first, worked like a charm, after that I tried the iPhone, worked also. No matter if I used the IP address in the browser, the hostname or the QR code, it all worked fine. 

I didn’t change a thing, so I guess it was just some kind of hiccup that day... Who knows... 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
